examples/tgn.py

```diff
@@ -26,14 +27,17 @@
 
 path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'JODIE')
 dataset = JODIEDataset(path, name='wikipedia')
-data = dataset[0].to(device)
+data = dataset.data
```

Comment:
I think we should reset to `dataset[0].to(device)`.

Reply:
Thanks for your comment. I think the `data = dataset[0]` pattern is quiet common in PyG. Note that `dataset.data` will give you the `data` of all examples inside `dataset` and leads to unexpected bugs whenever `len(dataset) > 1`.

Putting all `data` on GPU should definitely be faster than doing this for every batch (as long as the `dataset` fits on GPU). How about we add a comment here that tries to clarify this?